streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java

```diff
@@ -153,7 +153,7 @@ public void maybeNotifyTopologyVersionWaiters(final String threadName) {
                     if (threadVersions.values().stream().allMatch(t -> t >= topologyVersionWaitersVersion)) {
                         topologyVersionWaiters.future.complete(null);
                         iterator.remove();
-                        log.info("Thread {} is now on topology version {}", threadName, topologyVersionWaiters.topologyVersion);
+                        log.info("All threads are now on topology version {}", topologyVersionWaiters.topologyVersion);
```

Comment:
Sorry, some comments to this change:
I can imagine after this PR, there will be some log output in `maybeNotifyTopologyVersionWaitersAndUpdateThreadsTopologyVersion` like this (if the `version.activeTopologyWaiters` has 3 waiters):
```
[2021-12-15 08:52:59,397] INFO All threads are now on topology version 1
[2021-12-15 08:52:59,397] INFO All threads are now on topology version 1
[2021-12-15 08:52:59,397] INFO All threads are now on topology version 1
...
```
I don't think the log is helpful. Could we add some other info here? Like: `All threads are now on topology version {}, removing the waiter {}, topologyVersionWaiters.topologyVersion, topologyVersionWaiters.future)`. WDYT?

Reply:
this log line only prints once not once per thread. each waiter also has a different version

Reply:
I see. Thank you.